src/pages/Clients/Eth2/Lighthouse.tsx

```diff
@@ -56,12 +57,8 @@ export const LighthouseDetails = ({ shortened }: { shortened?: boolean }) => (
       <SectionTitle level={2} className="mb5">
         Installation
       </SectionTitle>
-      <Link
-        external
-        primary
-        to="https://lighthouse-book.sigmaprime.io/installation.html"
-      >
-        https://lighthouse-book.sigmaprime.io/installation.html
+      <Link external primary to={LIGHTHOUSE_INSTALLATION_URL} withArrow>
```

Comment:
I think the husky linter changes the format automatically when I commited. Do I have to force revert the format?

Reply:
Husky is very aggressive, and I'm always nervous when I see all so many files being changed when committing. That said, it does keep the formatting consistent and I've made peace with the git-history having many such changes. (If you force revert now, it will just update the ext time someone commits.)